openvino_tensorflow/executable.cc

```diff
@@ -32,12 +32,12 @@ Executable::Executable(shared_ptr<Function> func, string device,
       m_trivial_fn{nullptr},
       m_function(func) {
   OVTF_VLOG(2) << "Checking for unsupported ops";
-  const auto& opset = ngraph::get_opset7();
+  const auto& opset = ov::get_opset8();
   for (const auto& node : func->get_ops()) {
     if (!opset.contains_op_type(node.get())) {
```

Comment:
@ck-intel @cavusmustafa Updating to opset8 will not solve the same issue in the future: if TF FE will generate operations from opset9 and so on. Maybe this check is not really needed, because it shouldn't matter which opset TF FE generate operations from.